hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/http/RatisDropwizardExports.java

```diff
@@ -54,6 +54,13 @@ public static void registerRatisMetricReporters(
         r2 -> deregisterDropwizard(r2, ratisMetricsMap));
   }
 
+  public static void clear(Map<String, RatisDropwizardExports>
+                               ratisMetricsMap) {
+    MetricRegistries.global().clear();
+    CollectorRegistry.defaultRegistry.clear();
+    ratisMetricsMap.clear();
+  }
```

Comment:
Hey @szetszwo, could you help to review this Ratis metrics un-register logic ?

Reply:
+1 the change looks good.